assets/xml/objects/object_jso.xml

```diff
@@ -1,122 +1,147 @@
-        <Limb Name="object_jso_Standardlimb_00A8D0" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_01" Offset="0xA8D0" />
-        <Limb Name="object_jso_Standardlimb_00A8DC" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_02" Offset="0xA8DC" />
-        <Limb Name="object_jso_Standardlimb_00A8E8" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_03" Offset="0xA8E8" />
-        <Limb Name="object_jso_Standardlimb_00A8F4" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_04" Offset="0xA8F4" />
-        <Limb Name="object_jso_Standardlimb_00A900" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_05" Offset="0xA900" />
-        <Limb Name="object_jso_Standardlimb_00A90C" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_06" Offset="0xA90C" />
-        <Limb Name="object_jso_Standardlimb_00A918" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_07" Offset="0xA918" />
-        <Limb Name="object_jso_Standardlimb_00A924" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_08" Offset="0xA924" />
-        <Limb Name="object_jso_Standardlimb_00A930" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_09" Offset="0xA930" />
-        <Limb Name="object_jso_Standardlimb_00A93C" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_0A" Offset="0xA93C" />
-        <Limb Name="object_jso_Standardlimb_00A948" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_0B" Offset="0xA948" />
-        <Limb Name="object_jso_Standardlimb_00A954" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_0C" Offset="0xA954" />
-        <Limb Name="object_jso_Standardlimb_00A960" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_0D" Offset="0xA960" />
-        <Limb Name="object_jso_Standardlimb_00A96C" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_0E" Offset="0xA96C" />
-        <Limb Name="object_jso_Standardlimb_00A978" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_0F" Offset="0xA978" />
-        <Limb Name="object_jso_Standardlimb_00A984" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_10" Offset="0xA984" />
-        <Limb Name="object_jso_Standardlimb_00A990" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_11" Offset="0xA990" />
-        <Limb Name="object_jso_Standardlimb_00A99C" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_12" Offset="0xA99C" />
-        <Limb Name="object_jso_Standardlimb_00A9A8" Type="Standard" EnumName="OBJECT_JSO_2_LIMB_13" Offset="0xA9A8" />
-        <Skeleton Name="object_jso_Skel_00AA00" Type="Flex" LimbType="Standard" LimbNone="OBJECT_JSO_2_LIMB_NONE" LimbMax="OBJECT_JSO_2_LIMB_MAX" EnumName="ObjectJso2Limb" Offset="0xAA00" />
-        <Animation Name="object_jso_Anim_00AE80" Offset="0xAE80" /> <!-- Original name is "jso_wait" -->
-        <Animation Name="object_jso_Anim_00B1DC" Offset="0xB1DC" /> <!-- Original name is "jso_waitD" -->
-        <Animation Name="object_jso_Anim_00B5F0" Offset="0xB5F0" /> <!-- Original name is "jso_walk" -->
+        <!-- Unused Empty Texture Animations -->
+        <TextureAnimation Name="gGaroEmptyTexAnim1" Offset="0x0" />
+        <TextureAnimation Name="gGaroEmptyTexAnim2" Offset="0x10" />
```

Comment:
```suggestion
        <TextureAnimation Name="gGaroEmpty1TexAnim" Offset="0x0" />
        <TextureAnimation Name="gGaroEmpty2TexAnim" Offset="0x10" />
```
Not sure if there's a standard since `TextureAnimation` are more rare, but I think it's nice to end the name with the assent type, and leave the numbering (1/2) earlier in the name?

Reply:
Oh right, I completely forgot about this! Thanks for the reminder. I'll fix it.

Reply:
Number before Tex makes the most sense, even if it reads worse